custom/up_nrhm/filters.py

```diff
@@ -105,7 +105,7 @@ def get_hierarchy(self):
 
 class SampleFormatFilter(BaseSingleOptionFilter):
     slug = 'sf'
-    label = 'Report type'
+    label = ugettext_noop('Report type')
```

Comment:
These should all be using `ugettext_lazy`. See http://commcare-hq.readthedocs.org/en/latest/translations.html for details.